build/easy-select.js

```diff
@@ -103,6 +103,7 @@
         // exit if is native select
         if(this.config.nativeSelect){
             this.wrapper.addClass(names.wrapperNativeSelectClass);
+            this.select.attr("tabindex", "-1");
```

Comment:
Please convert this line into VanillaJs, and also add some comments for better maintenance.

Reply:
Actually, just add some comments, this could be jQuery for now.

Reply:
Your comment should be "skip focus on native select tag"